ee_tests/src/quickstart_deployments.spec.ts

```diff
@@ -128,93 +136,13 @@ describe('Creating new quickstart in OSIO', () => {
-    }
-
-    if (testRun(strategy)) {
-      support.info('Verifying application\'s run environment');
-      let environment = environments[Environment.RUN];
-
-      expect(environment.isReady()).toBeTruthy('run environment pod is ready');
-      expect(environment.getStatus()).toBe(DeploymentStatus.OK, 'run environment status');
-      expect(environment.getVersion()).toBe('1.0.1', 'run environment version');
-      expect(environment.getPodsCount()).toBe(1, 'number of pods on run environment');
-      // TODO this does not work correctly at the moment
-      // expect(await environment.getRunningPodsCount()).toBe(1, 'number of running pods on run environment');
-    }
-
-    support.info('Verifying resources usage');
-
-    let data = await spaceDeploymentsPage.getResourceUsageData();
-    expect(data.length).toBe(2, 'there should be stage and prod environment');
-
-    if (testStage(strategy)) {
-      support.info('Verifying stage environment resource usage');
-
-      let stageData = data[Environment.STAGE];
-      let stageDataItems = await stageData.getItems();
-      expect(stageDataItems.length).toBe(2, 'there should be 2 resource usage data items');
-
-      let cpu = stageDataItems[0];
-      expect(cpu.getActualValue()).toBeGreaterThan(0, 'the actual cpu usage data has to be greater than 0');
-      expect(cpu.getActualValue()).toBeLessThanOrEqual(cpu.getMaximumValue(), 'the actual cpu usage data has to be lower or equal to maximum');
-
-      let memory = stageDataItems[1];
-      expect(memory.getActualValue()).toBeGreaterThan(0, 'the actual memory usage data has to be greater than 0');
-      expect(memory.getActualValue()).toBeLessThanOrEqual(memory.getMaximumValue(), 'the actual memory usage data has to be lower or equal to maximum');
-    }
-
-    if (testRun(strategy)) {
-      support.info('Verifying run environment resource usage');
 
-      let stageData = data[Environment.RUN];
-      let stageDataItems = await stageData.getItems();
-      expect(stageDataItems.length).toBe(2, 'there should be 2 resource usage data items');
-
-      let cpu = stageDataItems[0];
-      expect(cpu.getActualValue()).toBeGreaterThan(0, 'the actual cpu usage data has to be greater than 0');
-      expect(cpu.getActualValue()).toBeLessThanOrEqual(cpu.getMaximumValue(), 'the actual cpu usage data has to be lower or equal to maximum');
-
-      let memory = stageDataItems[1];
-      expect(memory.getActualValue()).toBeGreaterThan(0, 'the actual memory usage data has to be greater than 0');
-      expect(memory.getActualValue()).toBeLessThanOrEqual(memory.getMaximumValue(), 'the actual memory usage data has to be lower or equal to maximum');
-    }
+    await deploymentsSpecStrategy.showDeploymentsScreen();
+    let application = await deploymentsSpecStrategy.verifyApplication();
+    await deploymentsSpecStrategy.verifyEnvironments(application);
+    await deploymentsSpecStrategy.verifyResourceUsage();
```

Comment:
I don't like the name... we are not verifying resource usage of a strategy but rather of application/deployment/pipeline.